### DISCUSSION TITLE
An Emacs interface for git-branchless

### DISCUSSION THREAD

**Discussion by Zetagon** [arxanas/git-branchless#687] [Show and tell]
Hello!

This tool looked interesting (especially the divergent development workflow!), but alas, there was no Emacs interface so trying it out was not an option.  I did the only sensible thing to do in that situation and hacked together an interface for Emacs!  https://git.sr.ht/~zetagon/git-branchless.el  Now I can safely try out the tool in question (and it feels pretty good to use!).

The interface isn't complete or super polished yet, but it is at least working for me so I thought I'd share.

Cheers

**Comment by bsima** [comment]
Wow you even have magit integration too! Wonderful! Great work!
